### ISSUE TITLE
Put package on melpa

### ISSUE THREAD

**Issue by codingquark** [coala/coala-emacs#4]
It is much much easier to have packages installed from [melpa](https://melpa.org/) or [marmalade](https://marmalade-repo.org/).

**Comment by sils** [comment]
Hey @codingquark do you know how to do this? I'm not sure if we have someone who has knowledge of that :)

**Comment by sils** [comment]
Thanks for your interest btw.!

**Comment by codingquark** [comment]
Unfortunately, I do not know. But I'll find out. :)

**Comment by lepisma** [comment]
I too need this on some package archive. Should I put it up on melpa?

**Comment by sils** [comment]
@lepisma that'd be awesome, can you give us access somehow so we can maintain the package?

**Comment by lepisma** [comment]
It will just be a simple pull request to melpa (like [this](https://github.com/lepisma/melpa/commit/c0d028002256fa213b186dcb02d205779192c249) commit). This repository will be the go-to source. Also, a release tag will be useful.

**Comment by sils** [comment]
Awesome! I'll just fix twoish things here then make a release tag for you within an hour or so.

**Comment by sils** [comment]
@lepisma I made a fix and tagged latest master with 0.9.0

**Comment by lepisma** [comment]
@sils if you merge #10 , could you also move the release tag to the commit with version in header `;; Version: 0.9.0`? Thanks

**Comment by sils** [comment]
uhm moving a tag isn't usually good practice, it's more of a bugfix release so I made https://github.com/coala/coala-emacs/releases/tag/0.9.1 after which I realized that you just declared the version as 0.9.0 in your PR.  So let's declare the version as 0.9.2 and release it as 0.9.2?

**Comment by lepisma** [comment]
Actually, there are a couple more issues before moving ahead. The checker is not working as of now.
- There are no modes (`python-mode`, `js-mode` etc.) set for flycheck to bind this checker with
- It doesn't add itself to the list of checker `'flycheck-checkers` or provide a setup function for the same
- Even after fixing these, there is some issue with the parsing.

I will formally file issues once I get free and pick out the problem in these couple of days. I guess tags should follow these few fixes?

**Comment by sils** [comment]
yep we can tag anything but moving a tag is very bad practice, releases are immutable. We can do as many bugfix releases as we want, I'd restrict feature releases to be in synch with coala, i.e. we should wait til coala 0.10 (~2 weeks) if we want to add features.

**Comment by sils** [comment]
Short: I'd first fix everything, then release 0.9.2

**Comment by sils** [comment]
Btw. 0.10 was just released, feel free to upgrade (or not, API didn't change)

**Comment by 9r0x** [comment]
I can't find it on Melpa, did you guys successfully put it there?
